cypress-io/cypress
# chore(app): add google beta icon

- Closes #25968 

### Additional details
- Currently the icon for the regular "Chrome" browser is being used for the "Chrome Beta" browser. This can make the UI look like it is duplicating browsers when it should be showing a unique list

### Steps to test
- `yarn workspace @packages/app dev`
- Launch the `LayeredBrowserIcons.cy.tsx` test
- View the last row of icons and you will see `Chrome`, `Chrome Canary` and `Chrome Beta` icons

### How has the user experience changed?
![Screen Shot 2023-03-06 at 2 20 43 PM](https://user-images.githubusercontent.com/3605268/223209290-8887e163-c6f5-421b-b151-35f25af3ba6c.png)

### PR Tasks
